packages/rich-text/src/create.js

```diff
@@ -274,14 +275,27 @@ function createFromElement( {
 			}
 		}
 
-		const value = createFromElement( {
-			element: node,
-			range,
-			removeNode,
-			unwrapNode,
-			filterString,
-			removeAttribute,
-		} );
+		if ( node.nodeName === 'UL' || node.nodeName === 'OL' ) {
+			value = createFromMultilineElement( {
```

Comment:
Are `UL` and `OL` the only multiline tags we support?

If not, where do we account for the others?

Reply:
I changed this so that the wrapper tags are defined by `RichText` instead.

Reply:
I guess my main question / concern would be: Are there going to be similar issues if someone uses any other multiline tag because we only have built-in correcting for the list usage?

Reply:
I can't see why there would be issues. They could use a different `multilineTag` and `multilineWrapperTags`. The value no longer has any built in logic for lists after your suggested change.